contracts/ModuleRegistry.sol

```diff
@@ -67,14 +75,47 @@ contract ModuleRegistry is IModuleRegistry, Pausable, RegistryUpdater, ReclaimTo
     function registerModule(address _moduleFactory) external whenNotPaused returns(bool) {
         require(registry[_moduleFactory] == 0, "Module factory should not be pre-registered");
         IModuleFactory moduleFactory = IModuleFactory(_moduleFactory);
-        require(moduleFactory.getType() != 0, "Factory type should not equal to 0");
-        registry[_moduleFactory] = moduleFactory.getType();
-        moduleList[moduleFactory.getType()].push(_moduleFactory);
+        uint8 kind = moduleFactory.getType();
```

Comment:
instead of "kind" use "type"

Reply:
type is a reserved keyword [documentation](https://solidity.readthedocs.io/en/v0.4.21/miscellaneous.html#reserved-keywords)

Reply:
I mean, moduleFactoryType or _type, don't want to be introducing different names.

Reply:
okay fixing